### DISCUSSION TITLE
How do disable references to `xxx`?

### DISCUSSION THREAD

**Discussion by hungify2022** [zed-industries/zed#10305] [Feature Requests]
How do we prevent clicking on attribute HTML tags or variables, etc?

Look at the video below, is there any configuration to disable it? (vscode only shows underline and unclickable)

https://github.com/zed-industries/zed/assets/126773143/da93e241-e7ec-465c-a8ee-18fbe6450c9b

**Comment by JosephTLyons** [comment]
Hey @hungify2022, there isn't curretnly a setting to disable `find all references` from being triggered when holding down `cmd` and clicking on some item in the editor. I'm curious to hear about you wanting to disable it - is it getting in the way of your workflow?

**Reply by hungify2022** [reply]
Thank you for getting back to me.
Sometimes I accidentally click on the wrong thing, which causes a lot of new tabs to open up.
Clicking on the variable creates some noise. Opening a new tab for `References to envVariables` is unnecessary.
Perhaps it is clickable on some stuff rather than everything.


![CleanShot 2024-04-10 at 08 57 28@2x](https://github.com/zed-industries/zed/assets/126773143/87b87736-939a-49ba-945f-782b5382b557)

**Reply by danjfaulkner** [reply]
I would like to be able to disable this too, just went back to vscode because I keep accidentally opening reference

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
